IOS_SETUP.md

```diff
@@ -32,6 +32,28 @@ Additionally, you must configure the background task identifiers. The default id
 </plist>
 ```
 
+
+You MUST amend your `AppDelegate.swift` and `Info.plist` file to register your task ID.
+
+- AppDelegate.swift
+```
+WorkmanagerPlugin.registerTask(withIdentifier: "task-identifier")
+```
+
+- Info.plist
+```
+<key>BGTaskSchedulerPermittedIdentifiers</key>
```

Comment:
please format this a bit more consistent. Also, this is discussed in line 28 already. Please simplify.